### PR TITLE
Switching to a one day workshop,

### PR DESCRIPTION
**Pull request by bwlang**

### CODE REVIEW

File: _events/2021-09-15-sars-cov-2-training.md [hunk 1]
```diff
@@ -47,44 +50,55 @@ During the workshop, there will be live support in chat and live Q&A sessions, i
 
 # Program & Material
 
-This workshop is virtual and asynchronous. Training sessions are pre-recorded and the material provided in advance in the program below.
 
-Whenever you're ready to get started, you can access the material by clicking in the program on the different icons:
+This workshop is has both live and virtual/asynchronous components. Training sessions are pre-recorded with most materials provided in advance in the program below.
```
Comment (beatrizserrano) [inline _events/2021-09-15-sars-cov-2-training.md:54]:
```suggestion
This workshop has both live and virtual/asynchronous components. Training sessions are pre-recorded with most materials provided in advance in the program below.
```

**Review by beatrizserrano — COMMENTED** [review]
thanks @bwlang! I added some minor comments to fix typos :)

**Comment by bwlang** [comment]
Ok - let’s push the date back - how about Wednesday November 3?

**Comment by bwlang** [comment]
Ok - I think we're all set.
Does the figure look ok?
Anything else we need before proceeding?

**Review by beatrizserrano — APPROVED** [review]

**Pull request merged**